github/event_types.go

```diff
@@ -480,19 +480,24 @@ type PublicEvent struct {
 // GitHub API docs: https://developer.github.com/v3/activity/events/types/#pullrequestevent
 type PullRequestEvent struct {
 	// Action is the action that was performed. Possible values are: "assigned",
-	// "unassigned", "labeled", "unlabeled", "opened", "closed", or "reopened",
-	// "synchronize", "edited". If the action is "closed" and the merged key is false,
+	// "unassigned","review_requested","review_request_removed" "labeled", "unlabeled",
+	// "opened", "closed", or "reopened","synchronize", "edited".
```

Comment:
Add space between `"reopened","synchronize"`, and get rid of `or`, it's not needed.